pyanaconda/ui/gui/spokes/installation_source.py

```diff
@@ -811,93 +810,36 @@ def _get_selected_partition(self):
             return store[selected][0]
 
     # Input checks
-
-    # This method is shared by the checks on urlEntry and repoUrlEntry
-    def _check_url(self, inputcheck, combo):
+    def _check_url_entry(self, inputcheck):
```

Comment:
It might be useful to extend the commit message with the part where `_check_url` was split into two others which both use the new helper. It wasn't exactly clear from the diff...